server/services/rflink/lib/utils/colors.js

```diff
@@ -0,0 +1,59 @@
+const c2xterm = require('color2xterm');
+const { intToHex } = require('../../../../utils/colors');
+
+/**
+ * @description Convert an integer to an 8 bits hexa code.
+ * @param {string} intColor - A number.
+ * @returns {string} - The converted hexa code.
+ * @example
+ * to8bitsHex(15022);
+ * console.log(int === FF);
+ */
+function to8bitsHex(intColor) {
```

Comment:
We already have a color utils. Can you move this in `server/utils/colors.js` file please?